### DISCUSSION TITLE
Jarvis 3.0 Fernzugriff

### DISCUSSION THREAD

**Discussion by hardl1** [Zefau/ioBroker.jarvis#987] [General]
Es gelingt mir nicht eine Jarvis 3.0-70 Visualisierung über ioBroker.net oder .pro aufzurufen. 
Jarvis wird nicht unter Anwendungen angezeigt.
Im pro -> Admin -> Jarvis bleibt der Bildschirm leer und connection delayed.

Beide Varianten funktionieren mit VIS problemlos. 

Mache ich was falsch oder funktioniert das mit Jarvis (noch) nicht?

**Comment by Zefau** [comment]
Wird offiziell nicht unterstützt. Ich hab nichts mit ioBroker.net / ioBroker.pro zu tun.

**Comment by Zefau** [comment]
Hierfür kommt https://github.com/Zefau/ioBroker.jarvis/issues/71

**Comment by hardl1** [comment]
Gibt es dazu einen Zeitvorstellung?

**Comment by Zefau** [comment]
Erstmal muss ich v3 releasen - voraussichtlich in Q4. Danach dann. Vote es gerne hoch, damit es auf die Roadmap kommt.

**Reply by Zefau** [reply]
Die Roadmap ergibt sich u.a. durch die Votes der Feature Requests, siehe https://github.com/Zefau/ioBroker.jarvis/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions+

**Comment by Moo1988** [comment]
Is there a solution now? I don't understand why there is visualization software that you can't achieve using home remedies. I think Jarvis is really great, but without remote access it's useless for most people... it's a shame

**Reply by mcuiobroker** [reply]
I always use VPN Wireguard. You just activate the VPN and you can use whereever you want?
